index.js

```diff
@@ -96,7 +101,10 @@ class Multifeed extends Nanoresource {
     this._handlers.fetchFeeds((err, infos) => {
       if (err) return cb(err)
       for (const info of infos) {
-        const feed = this._corestore.get({ key: info.key })
+        const feed = this._corestore.get({
+          key: info.key,
+          ...this._opts
```

Comment:
perhaps this should be a custom option `coreOpts` so you can pass hypercore specific opts for reinitializing cores?

Reply:
Ideally in future we're going to want to be more specific about the ways we initialize feeds in multifeed, for example, when different feeds serve different purposes. At the moment, we have no way of adding specific parameters to the reinitializing of feeds.